docker-compose.yml

```
@@ -9,10 +9,9 @@ services:
     env_file:
       - .env
     
-    # or 
-    # environment:
-      #- BACKEND_PORT=7001   # if you change the port, make sure to also change the VITE_WS_BACKEND_URL at frontend/.env.local
-      # - OPENAI_API_KEY=your_openai_api_key
+    environment:
+      - BACKEND_PORT=7001 # if you change the port, make sure to also change the VITE_WS_BACKEND_URL at frontend/.env.local
```

it would be a good practice to make BACKEND_PORT a variable as well
```suggestion
      -  - BACKEND_PORT=${BACKEND_PORT:-7001} # if you change the port, make sure to also change the VITE_WS_BACKEND_URL at frontend/.env.local
```